_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+# Lecture 8 : Causal Discovery and Inference
```

Comment:
```suggestion
---
layout: distill
title: "Lecture 8: Causal Discovery and Inference"
description: <insert description here>
date: <insert date here>

lecturers:
  - name: Eric Xing
    url: "https://www.cs.cmu.edu/~epxing/"

authors:
  - name: Allan Wang
    url: "#"  # optional URL to the author's homepage
  - name: Binghao Deng
    url: "#"
  - name: Weizhao Shao
    url: "#"
  - name: Zhuoran Liu

editors:
  - name: Lisa Lee
    url: "http://leelisa.com"

abstract: >
---
```
Please follow the lecture note template. You'll need to update the description, date, and author names in the above suggestion. For an example, see https://raw.githubusercontent.com/sailinglab/pgm-spring-2019/master/_posts/2019-01-23-lecture-03.md